install-and-configure/install/first-time-user-guide.md

```diff
@@ -1,49 +1,37 @@
+By default, Kubecost will detect the Cloud Service Provider (CSP) of the cluster where it is installed and pull list prices for nodes, storage, network transfer, and LoadBalancers. For Azure, AWS, and GCP- this works without any additional Kubecost configuration.&#x20;
 
-New users should seek to integrate any and all CSPs they use into Kubecost. For an overview of cloud integrations and getting started, see our [Cloud Billing Integrations](/install-and-configure/install/cloud-integration/README.md) doc. Once you have completed all necessary integrations, return to this article.
+However, many organizations have discounts with cloud service providers (CSPs). Kubecost supports `cloud-integrations` which pull CSP billing information to reconcile costs to reflect enterprise discounts, Spot market prices, commitment discounts, and more.&#x20;
 
-Due to the frequency of updates from providers, it can take anywhere from 24 to 48 hours to see adjusted costs.
+For an overview of cloud integrations and getting started, see our [Cloud Billing Integrations](/install-and-configure/install/cloud-integration/README.md) doc. Once you have completed all necessary integrations, return to this article.
 
-## Step 2: Review your data
+Due to the frequency of billing updates from providers, it can take anywhere from 24 to 48 hours to see adjusted costs.
 
-Now that your base install and CSP integrations are complete, it's time to determine the accuracy against your cloud bill. Based on different methods of cost aggregation, Kubecost should assess your billing data within a 3-5% margin of error.
+## Currency Types
 
-You can also configure your currency display type. Kubecost supports the following currency types: USD, AUD, BRL, CAD, CHF, CNY, DKK, EUR, GBP, IDR, INR, JPY, NOK, PLN, and SEK. Kubecost does *not* perform any currency conversion when switching currency types; it is for display purposes, therefore you should ideally match your currency type to the type in your original cloud bill(s).
+Kubecost uses USD by default, though you can also configure your currency display type. Kubecost supports the following currency types: USD, AUD, BRL, CAD, CHF, CNY, DKK, EUR, GBP, IDR, INR, JPY, NOK, PLN, and SEK. Kubecost does *not* perform any currency conversion when switching currency types; it is for display purposes, therefore you should ideally match your currency type to your cloud billing.
 
 Currency type can only be changed via a [`helm` upgrade to your *values.yaml*](/install-and-configure/install/helm-install-params.md), using the flag `.Values.kubecostProductConfigs.currencyCode`. For example, if you needed to convert your currency type to EUR, you would add the following to your `helm` command:
 
-```
+```sh
 --set kubecostProductConfigs.currencyCode=EUR
 ```
 
-### Monitoring your cost billing
```

Comment:
This is covered below and doesn't really flow based on the topics above and below.